src/jdk.incubator.code/share/classes/jdk/incubator/code/internal/QuotedHelper.java

```diff
@@ -37,4 +37,7 @@ public static Quoted makeQuoted(MethodHandles.Lookup lookup, String opText, Obje
         FuncOp op = (FuncOp)OpParser.fromStringOfFuncOp(opText);
         return (Quoted)Interpreter.invoke(lookup, op, args);
     }
+    public static Quoted makeQuoted(MethodHandles.Lookup lookup, FuncOp op, Object[] args) {
```

Comment:
Can the prior method accepting a string be removed?

Reply:
I think so